tools/glyph-studio/py/section_propagate_eval.py

```
@@ -0,0 +1,73 @@
+
+import numpy as np
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+from glyphstudio.section_propagate import evaluate_cross_receipt  # noqa: E402
+
+
+def load_words(snapshot: str, section_map: dict):
+    import chromadb
+
+    client = chromadb.PersistentClient(path=snapshot)
+    coll = [c for c in client.list_collections() if c.name == "words"][0]
```

Ruff (0.15.20)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)